src/roiextractors/extractors/tiffimagingextractors/scanimagetiffimagingextractor.py

```diff
@@ -651,6 +682,31 @@ def get_plane_extractor(self, plane_index: int) -> ImagingExtractor:
 
         return sliced_extractor
 
+    @staticmethod
+    def get_slices_per_sample(file_path: PathType) -> int:
```

Comment:
Couple questions:
1. Why are you defining this here as a staticmethod if it only gets used in the tests?
2. What is the point of introducing a new term (slices per sample) if it's just frames per slice?

Reply:
Ah, thanks for pointing out the bad term. I mixed that up badly. This is me being tired and dumb : P. I will correct.

The static method is to enable the workflow B here:

https://github.com/catalystneuro/neuroconv/issues/1260#issue-2921052096

1) First you find out the number of frames_per_slice. 
2) You instantiate one extractor per sample.
3) You write each of them as a separate Series.

Reply:
I see, so it's analogous to the `get_available_channels` and `get_available_planes` methods from the older extractors?